tests/subsys/disk/stm32_sdmmc/src/main.c

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2020 Amarula Solutions
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <devicetree.h>
+
+#if !DT_HAS_NODE(DT_INST(0, st_stm32_sdmmc))
+#error stm32 sdmmc disk access not enabled
+#endif
+
+void test_main(void)
+{
```

Comment:
Thanks for proposing this sample.
Though, can you provide an even quite basic example on how to exercise APIs?

Reply:
For my project I just use it through the file system API. Would it be fine to enable FAT and then in the sample just mount it and write something?
Or should I directly use the disk API?

Reply:
> Would it be fine to enable FAT and then in the sample just mount it and write something?

@anthonybrandon would you manage `samples/subsys/fs/fat_fs` compiling/running on that board ?